configure.sh

```diff
@@ -43,6 +43,8 @@ function platform() {
 		PLATFORM='linux'
 	elif [[ "$unamestr" == 'Darwin' ]]; then
 		PLATFORM='mac'
+	elif [[ "$unamestr" =~ 'MINGW' ]]; then
+		PLATFORM='windows'
 	fi
 }
 
```

Comment:
nit: it would be great if instead of using a global variable we returned a value from `platform`. Then passed it to `downloadJar` like: `downloadJar(getPlatform())`